src/main/java/step2/Calculator.java

```diff
@@ -0,0 +1,33 @@
+package step2;
+
+public class Calculator {
+
+  private final static String SPLIT_SPACE = " ";
+
+  public int calculate(String input) {
```

Comment:
```suggestion
  public static int calculate(String input) {
```
인스턴스 변수를 사용하지 않으니 static으로 만들어도 문제 없겠네요
다른 메서드도 마찬가지입니다